src/backend/templates/rooms.hbs

```diff
@@ -132,12 +134,15 @@
               {{/if}}
                       </p>
 		             	{{/speakers_list}}
+                  </div>
+                  <div class="col-md-6" style="padding:0px">
```

Comment:
@oanarosca Please add all styling in application.scss. We are trying to remove the inline styling to make code more readable.